vite.config.js

```diff
@@ -0,0 +1,26 @@
+import { defineConfig } from "vite";
+import react from "@vitejs/plugin-react";
+import { NodeGlobalsPolyfillPlugin } from "@esbuild-plugins/node-globals-polyfill";
+import { NodeModulesPolyfillPlugin } from "@esbuild-plugins/node-modules-polyfill";
+
+// https://vitejs.dev/config/
+export default defineConfig({
+  plugins: [react()],
+  build: {
+    outDir: "build",
+  },
+  define: {
+    VITE_APP_VERSION: JSON.stringify(process.env.npm_package_version),
```

Comment:
Since I'm new to Vite I wasn't exactly sure how different environment variables played together. Let me see if I can get it into the .env file properly.